src/vs/platform/windows/electron-main/windowsMainService.ts

```diff
@@ -75,6 +75,8 @@ interface IOpenBrowserWindowOptions {
 	readonly windowToUse?: ICodeWindow;
 
 	readonly emptyWindowBackupInfo?: IEmptyWindowBackupInfo;
+
+	readonly profile?: ProfileOptions;
```

Comment:
This is not really the options for a profile but the name of the profile right?

Reply:
For now it is just name of the profile, but going forward I want to include other configuration options like `templateUrl`, `transient` etc.,